lib/elixir/lib/file.ex

```diff
@@ -246,6 +246,18 @@ defmodule File do
     end
   end
 
+ @doc """
+ +  Returns `binary`, which is part of the contents of `path` starting at `start` and of length
+ +  `bytes`
+ +  """
+ +  @spec read(Path.t, pos_integer, pos_integer) :: binary 
+ +  def read(path, start, bytes) do
+ +    {ok, f} = :file.open(path, [:binary])
```

Comment:
`ok` is not an Elixir atom, it's a variable. You probably meant `:ok` here (and below).